### PR TITLE
Add support to kubernetes namespace

### PR DESCRIPTION
**Pull request by wsilva**
## Problem

Permission issues on GCP due to lack of namespace support

## Solution

Add namespace handling to terraform module

---

cc @jesusvazquez

### CODE REVIEW

File: google-cloud/sql/inputs.tf [hunk 1]
```diff
@@ -87,3 +87,8 @@ variable "project" {
   description = "Name of gcp project"
   type        = "string"
 }
+
+variable "namespace" {
+  description = "Name of kubernetes namespace"
+  type        = "string"
```
Comment (jesusvazquez) [inline google-cloud/sql/inputs.tf]: Default this to prometheus,, just like mysql scraper does.

**Comment by jesusvazquez** [comment]
Looking good.

File: google-cloud/sql/inputs.tf (hunk 1 above)
Comment (wsilva) [reply google-cloud/sql/inputs.tf]: done

**Comment by wsilva** [comment]
Should I merge it?

**Pull request merged**